Wikipedia/Code/AnnouncementCollectionViewCell.swift

```diff
@@ -132,20 +142,9 @@ open class AnnouncementCollectionViewCell: CollectionViewCell {
             messageTextView.attributedText = nil
             return
         }
-        let attributedText = html.byAttributingHTML(with: .subheadline,
-                                                    boldWeight: .bold,
-                                                    matching: traitCollection,
-                                                    color: messageTextView.textColor,
-                                                    tagMapping: ["em": "i"], // em tags are generally italicized by default, match this behavior
-                                                    additionalTagAttributes: [
-            "u": [
-                NSAttributedString.Key.underlineColor: messageUnderlineColor,
-                NSAttributedString.Key.underlineStyle: NSNumber(value: NSUnderlineStyle.single.rawValue)
-            ],
-            "strong": [
-                NSAttributedString.Key.foregroundColor: messageEmphasisColor
-            ]
-        ])
+
+        let styles = HtmlUtils.Styles(font: WKFont.for(.subheadline, compatibleWith: traitCollection), boldFont: WKFont.for(.boldSubheadline, compatibleWith: traitCollection), italicsFont: WKFont.for(.italicSubheadline, compatibleWith: traitCollection), boldItalicsFont: WKFont.for(.boldItalicSubheadline, compatibleWith: traitCollection), color: theme.colors.primaryText, linkColor: theme.colors.link, strongColor: messageEmphasisColor, lineSpacing: 1)
```

Comment:
I remember the not implementing the `additionalTagAttributes` in the newer utils, since I wasn't sure how widely used it was. I'm good with leaving this as-is and seeing how design feels about it.